Source/Scene/PointCloudPostProcessor.js

```diff
@@ -170,49 +258,54 @@ define([
             blending : BlendingState.ALPHA_BLEND
         });
 
+        var blendUniformMap = {
+            pointCloud_colorTexture: function() {
+                return processor._colorTextures[1 - drawCommands.length % 2];
+            },
+            pointCloud_depthTexture: function() {
+                return processor._depthTextures[1 - drawCommands.length % 2];
+            }
+        };
+
         var blendCommand = context.createViewportQuadCommand(blendFS, {
-            uniformMap : uniformMaps[stagesLength % 2],
+            uniformMap : blendUniformMap,
             renderState : blendRenderState,
             pass : Pass.CESIUM_3D_TILE,
             owner : processor
         });
 
-        // TODO : may need to clear FBOs between each stage instead of just clearing the main FBO.
-        var clearCommand = new ClearCommand({
-            framebuffer : framebuffers[1],
-            color : new Color(0.0, 0.0, 0.0, 0.0),
-            depth : 1.0,
-            renderState : RenderState.fromCache(),
-            pass : Pass.CESIUM_3D_TILE,
-            owner : processor
-        });
+        var framebuffers = processor._framebuffers;
+        var clearCommands = new Array(4);
+        i = 0;
+        for (var name in framebuffers) {
```

Comment:
Throughout, it looks like `_framebuffers` is used as both an array and object.  Be consistent.  Perhaps this can be an array with constants in an enum for look ups.